examples/gno.land/r/demo/mail/predicates.gno

```diff
@@ -0,0 +1,5 @@
+package mail
+
+type Predicate interface {
+	Satisfy(mail *Mail) bool
+}
```

Comment:
Why separate out predicate logic from this file? Why not have all predicates grouped together under `predicates.gno` instead of having them in 5 different files?

Reply:
Well I admit it's bordering on the weird side. 
I am used to do whatever is convenient for me (I have bad eyesight and with vim I can have 10 splits very quickly if I want to).
But yes I must admit one file is more presentable.